pkg/otp/screenshot_others.go

```diff
@@ -1,5 +1,5 @@
-//go:build !(arm || arm64 || amd64 || 386) || !(linux || windows || darwin || freebsd || netbsd || openbsd)
-// +build !arm,!arm64,!amd64,!386 !linux,!windows,!darwin,!freebsd,!netbsd,!openbsd
+//go:build !(arm || arm64 || amd64 || 386) || !(linux || windows || freebsd || netbsd) || (darwin && !cgo)
```

Comment:
i suspect this won't build on (darwin && cgo) because of a duplicate ParseScreen definition.

In addition to including the _supported.go file, (darwin && cgo) is not (linux || windows || freebsd || netbsd).  So it would evaluate to true and include this file too.

This needs to be the inverse of the supported build tags:

```
//go:build !(arm || arm64 || amd64 || 386) || !(linux || windows || (cgo && darwin) || freebsd || netbsd)
```

or

```
//go:build !((arm || arm64 || amd64 || 386) && (linux || windows || (cgo && darwin) || freebsd || netbsd))
```

Reply:
Ah right, I was trying to tweak it since this is too complex for +build line, but actually since we're not supporting old Go versions, so we don't need to keep these anyway.